janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java

```diff
@@ -69,6 +70,11 @@
             ConfigOption.Type.MASKABLE,  String.class,
             IsolationLevel.REPEATABLE_READ.toString(), disallowEmpty(String.class));
 
+    public static final ConfigOption<Long> LOCK_TIMEOUT =
+        new ConfigOption<Long>(BERKELEY_NS,"lock-timeout",
+            "Lock timeout for all transactional and non-transactional operations",
```

Comment:
Description should mention that the units for the value is milliseconds, and that the default is 500ms and maximum is 75min according to the BerkeleyJE [LOCK_TIMEOUT docs](https://docs.oracle.com/cd/E17277_02/html/java/com/sleepycat/je/EnvironmentConfig.html#LOCK_TIMEOUT).